pkg/cmd/repo/setdefault/setdefault_test.go

```diff
@@ -179,13 +179,15 @@ func TestDefaultRun(t *testing.T) {
 		},
 		{
 			name: "view mode no current default",
+			tty:  false,
 			opts: SetDefaultOptions{ViewMode: true},
 			remotes: []*context.Remote{
 				{
 					Remote: &git.Remote{Name: "origin"},
 					Repo:   repo1,
 				},
 			},
+			wantStdout: "no default repository has been set; use `gh repo set-default` to select one\n",
```

Comment:
Since we are explicitly moving this from `stdout` to `stderr`, I think I'd prefer to see this as `wantStderr` rather than `wantStdout`